src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -36,20 +36,20 @@ folder InstallDir:\MSBuild\Current\Bin
   file source=$(X86BinPath)MSBuild.exe.config
   file source=$(TaskHostBinPath)MSBuildTaskHost.exe vs.file.ngenArchitecture=x86
   file source=$(TaskHostBinPath)MSBuildTaskHost.exe.config
-  file source=$(X86BinPath)System.Buffers.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Memory.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Text.Json.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)Microsoft.Bcl.AsyncInterfaces.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Text.Encodings.Web.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Threading.Tasks.Extensions.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.ValueTuple.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Numerics.Vectors.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Resources.Extensions.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Runtime.CompilerServices.Unsafe.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)System.Threading.Tasks.Dataflow.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
+  file source=$(X86BinPath)System.Buffers.dll vs.file.ngenApplications="[installDir]\MSBuild\Current\Bin\MSBuild.exe" vs.file.ngenArchitecture=all vs.file.ngenPriority=1
```

Comment:
Did you check with the Setup folks about adding `ngenPriority=1`? IIRC they had a very small list they wanted in that because it slows setup.

This is my only concern, PR LGTM so I'm signing off, but please check this before merging.

Reply:
@mjcavallari-zz can you please comment on this? These assemblies are loaded into the MSBuild process so both VS build (which spawns MSBuild satellite processes) and command line build are affected. As such I would prefer if they were NGENed synchronously during setup.

Reply:
To clarify, the before measurements were done after flushing the NGEN queue with `ngen eqi` so the bug is really addressed with `vs.file.ngenApplications` and not with priorities. We could remove `ngenPriority=1` if it turns out to be problematic.

Reply:
Setup doesn't currently block on any priority 1 NGEN images being generated (there's is a feature flag that switches this but its turned off however, see ShouldRunP1NgenSynchronously internally).

Rest of MSBuild appears to pri 1.

Reply:
Awesome, thank you for the insight! This should clear up the concern, then. Ready to go, @rainersigwald?

Reply:
Sounds good.